docs/content/ActionList.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: ActionList
+---
+
+An `ActionList` is a list of items which can be activated or selected. `ActionList` is the base component for many of our menu-type components, including `DropdownMenu` and `ActionMenu`.
+
+## Default example
+
+```javascript live noinline
+const Demo = () => {
+  return (
+    <ActionList
+      items={[
+        {text: 'New file'},
+        ActionList.Divider,
+        {text: 'Copy link'},
+        {text: 'Edit file'},
+        {text: 'Delete file', variant: 'danger'}
+      ]}
+    />
+  )
+}
+
+render(<Demo />)
+```
```

Comment:
I think we can simplify this a bit:

```suggestion
```jsx live
<ActionList
  items={[
    {text: 'New file'},
    ActionList.Divider,
    {text: 'Copy link'},
    {text: 'Edit file'},
    {text: 'Delete file', variant: 'danger'}
  ]}
/>
```
```

Reply:
Thanks! I’d emulated [Overlay.mdx](https://github.com/primer/components/blob/46c6b804ea38d29f58a4c1a92238a6a6c3729667/docs/content/Overlay.mdx#default-example) from https://github.com/primer/components/pull/1084—is the call to `render` unnecessary?

Reply:
Yeah, we only use the `render()` function for complex examples (e.g. examples that use hooks): https://primer.style/doctocat/usage/live-code#no-inline

Reply:
Cool, thanks for the info! ✨